### PR TITLE
Clarify CSS injection timing

### LINKED ISSUE #8914: Issue with "content_scripts": Clarify effect of "run_at" on CSS injection
MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts#run_at

#### What information was incorrect, unhelpful, or incomplete?
The documentation for the `run_at` key says:
> This option determines when the scripts specified in `js` are injected.

This is true, but it omits the fact that this option also controls when the CSS files specified in the `css` key are injected.

Later in the same section it is stated that:
> In all cases, files in `js` are injected after files in `css`.

Again, this is correct but a little unclear; it doesn't guarantee that the CSS won't be injected before the specified readyState.

#### Specific section or headline?
The `run_at` entry of the table documenting the keys that can be used in the extension manifest's content_scripts object.

#### What did you expect to see?
A clear statement that the CSS and JS files are injected (in that order) when the readyState specified by `run_at` is reached.

For example, see the [extensionTypes.RunAt](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extensionTypes/RunAt) page of the JavaScript APIs for WebExtensions documentation, which says:

> **extensionTypes.RunAt**
>
> The soonest that the JavaScript or CSS will be injected into the tab.

#### Did you test this? If so, how?
Yes, by making a css-only extension and confirming that the `run_at` key does control when the CSS is injected into the matched document.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/mozilla/add-ons/webextensions/manifest.json/content_scripts`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mozilla/add-ons/webextensions/manifest.json/content_scripts/index.html
* Last commit: https://github.com/mdn/content/commit/58d2649cd52f8c1d9baca86a1f02ca98b1fb2d09
* Document last modified: 2021-05-31T17:27:37.000Z

</details>

### PR DESCRIPTION
**Pull request by rcowsill**
Fixes #8914

The [content_scripts page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts) was unclear regarding the timing of CSS file injection. The [css](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts#css) section said they were injected before the DOM was loaded, and it wasn't clear that the [run_at](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts#run_at) key controlled when the CSS would be injected.

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_scripts/index.html [hunk 1]
```diff
@@ -78,9 +78,9 @@
    <td><a id="css"><code>css</code></a></td>
    <td><code>Array</code></td>
    <td>
-    <p>An array of paths, relative to <code>manifest.json</code>, referencing CSS files that will be injected into matching pages.</p>
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/content_scripts/index.html:81]: Please revert manifest.json to code formatting.

**Comment by zombie** [comment]
Just checked the code, these clarifications seem correct.
https://searchfox.org/mozilla-central/rev/fb7c66cb59/toolkit/components/extensions/ExtensionContent.jsm#560-561

Please address the comment from Richard, but otherwise LGTM.

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_scripts/index.html (hunk 1 above)
Comment (rcowsill) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/content_scripts/index.html:81]: Should I change the `js` section to match or leave as-is?

Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/content_scripts/index.html:81]: If you have the time, that would be great. Do you think you can also fix the 6 flaws (see the github-actions section)?

**Review by rebloor — APPROVED** [review]
Thanks @rcowsill These changes look good

**Pull request merged**